interfaceParams:
  IMAGE_WIDTH: 320
  IMAGE_HEIGHT: 240
  LANE_IMAGE_WIDTH: 224
  LANE_IMAGE_HEIGHT: 224
  WINDOW_WIDTH: 320
  WINDOW_HEIGHT: 400
  TEXT_BBOX: 5
